lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -70,66 +123,88 @@ public function do(FaceRecognitionContext $context) {
 		$fld = new \FaceLandmarkDetection("shape_predictor_5_face_landmarks.dat");
 		$fr = new \FaceRecognition("dlib_face_recognition_resnet_model_v1.dat");
 
-		// number of things that can go wrong below is too damn high:) be more defensive
 		foreach($images as $image) {
+			yield;
```

Comment:
moved yield to start. There was some good reason (not to be after finally), but I forgot (but, there was a reason!:)